TESTING.md

```diff
@@ -0,0 +1,99 @@
+# tardigrade-ci testing documentation
+
+Testing tardigrade-ci implementation with various setup on local environment.
+
+## Testing
+
+This project can be utilized one of two ways, via docker or via a Makefile include.
+
+### Via Docker (non-tardigrade-ci image)
+
+In a non-tardigrade-ci container, the second condition in Makefile.bootstrap ought to fail, resulting in the auto-init logic creating the tardigrade-ci/ directory.
+
+1. Run the following bash script or manually create a test-project directory with the following configurations:
+
+```bash
+#!/bin/bash
+
+mkdir test-project
+cd test-project
+git init
+echo "FROM plus3it/tardigrade-ci:0.8.0" > Dockerfile
+echo "SHELL := /bin/bash\n" > Makefile
+echo 'include $(shell test -f .tardigrade-ci || curl -sSL -o .tardigrade-ci "https://raw.githubusercontent.com/plus3it/tardigrade-ci/master/bootstrap/Makefile.bootstrap"; echo .tardigrade-ci)' >> Makefile
+echo '# tardigrade-ci' > .gitignore
+echo '.tardigrade-ci' >> .gitignore
+echo 'tardigrade-ci/' >> .gitignore
```

Comment:
I think we can add a couple steps to pull the docker containers we're referencing...

```suggestion
echo 'tardigrade-ci/' >> .gitignore

docker pull python:3
docker pull plus3it/tardigrade-ci:latest
```